tests/test_wave_01.py

```diff
@@ -30,9 +31,9 @@ def test_create_no_title_movie():
     new_movie = create_movie(movie_title, genre, rating)
 
     # Assert
-    assert new_movie is None
+    assert movie_title is None
```

Comment:
In this project, we shouldn't change tests that don't explicitly call for it. This change made this test pass but it also highlights something unaccounted for in the function you wrote that's being tested here. I'll leave a comment at that function with more details.